libs/sdk-ui-dashboard/src/model/events/alerts.ts

```diff
@@ -0,0 +1,202 @@
+        ctx,
+        correlationId,
+        payload: {
+            alert,
+            error,
+        },
+    };
+}
+
+//
+//
+//
+
+/**
+ * This event is emitted after the Kpi alert is successfully removed.
+ *
+ * @internal
+ */
+export interface DashboardAlertRemoveSuccess extends IDashboardEvent {
+    readonly type: "GDC.DASH/EVT.ALERTS.REMOVE.SUCCESS";
+    readonly payload: {
+        readonly alert: IWidgetAlert;
+    };
+}
+
+export function alertRemoveSuccess(
+    ctx: DashboardContext,
+    alert: IWidgetAlert,
+    correlationId?: string,
+): DashboardAlertRemoveSuccess {
+    return {
+        type: "GDC.DASH/EVT.ALERTS.REMOVE.SUCCESS",
+        ctx,
+        correlationId,
+        payload: {
+            alert,
+        },
+    };
+}
+
+//
+//
+//
+
+/**
+ * This event is emitted after the Kpi alert removing failed
+ *
+ * @internal
+ */
+export interface DashboardAlertRemoveError extends IDashboardEvent {
+    readonly type: "GDC.DASH/EVT.ALERTS.REMOVE.ERROR";
+    readonly payload: {
+        readonly alert: IWidgetAlert;
+        readonly error: any;
+    };
+}
+
+export function alertRemoveError(
+    ctx: DashboardContext,
+    alert: IWidgetAlert,
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+    error: any,
+    correlationId?: string,
+): DashboardAlertRemoveError {
+    return {
+        type: "GDC.DASH/EVT.ALERTS.REMOVE.ERROR",
+        ctx,
+        correlationId,
+        payload: {
+            alert,
+            error,
+        },
+    };
+}
+
+//
+//
+//
+
+/**
+ * This event is emitted after the Kpi alert is updated.
+ *
+ * @internal
+ */
+export interface DashboardAlertUpdateSuccess extends IDashboardEvent {
+    readonly type: "GDC.DASH/EVT.ALERTS.UPDATE.SUCCESS";
+    readonly payload: {
+        readonly updated: IWidgetAlert;
+    };
+}
+
+export function alertUpdateSuccess(
+    ctx: DashboardContext,
+    updated: IWidgetAlert,
+    correlationId?: string,
+): DashboardAlertUpdateSuccess {
+    return {
+        type: "GDC.DASH/EVT.ALERTS.UPDATE.SUCCESS",
+        ctx,
+        correlationId,
+        payload: {
+            updated,
+        },
+    };
+}
+
+//
+//
+//
+
+/**
+ * This event is emitted after the Kpi alert updating failed.
+ *
+ * @internal
+ */
+export interface DashboardAlertUpdateError extends IDashboardEvent {
+    readonly type: "GDC.DASH/EVT.ALERTS.UPDATE.ERROR";
+    readonly payload: {
+        readonly updated: IWidgetAlert;
+        readonly error: any;
+    };
+}
+
+export function alertUpdateError(
+    ctx: DashboardContext,
+    updated: IWidgetAlert,
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+    error: any,
+    correlationId?: string,
+): DashboardAlertUpdateError {
+    return {
+        type: "GDC.DASH/EVT.ALERTS.UPDATE.ERROR",
+        ctx,
+        correlationId,
+        payload: {
+            updated,
+            error,
+        },
+    };
+}
```

Comment:
this one is not needed; the generic event already can contain error etc. if we need to include detail about the command that failed in that generic event, we can enhance the event and include the entire cmd in it.